src/pages/code-insights.tsx

```diff
@@ -317,7 +317,36 @@ const CodeInsightsPage: FunctionComponent = () => (
                 product="code insights"
                 title={'Track what really matters\nto you and your team.'}
                 subtitle="Transform your code into a queryable database to create customizable, visual dashboards in seconds."
-                cta={<HubSpotForm masterFormName="contactEmail" chiliPiper={true} />}
+                cta={
+                    <div className="tw-text-center tw-flex-col md:tw-flex-row md:tw-flex">
```

Comment:
We've got a slightly different issue with this page after changing the alignment to `left`. You'll notice the "Request a demo" button not spanning the full width. This is because the `Hero` component has a `max-w-400` class applied on line `61` in `Hero.tsx`. I think we should change that to `350px` to match with the max width of these buttons and that would fix this issue.

We could also potentially remove `max-w-350` from each button since it would be redundant at that point since each button spans the full width of the parent with `w-100`, but that's not blocking; just a cleanup suggestion.

```suggestion
                    <div className="tw-text-left tw-flex-col md:tw-flex-row md:tw-flex">
```

![Screen Shot 2022-10-19 at 12 56 42 PM](https://user-images.githubusercontent.com/1733936/196757016-78eda75f-db24-45d3-a803-8764f6eb7cd0.png)

Reply:
I think if we change the `max-w-400` class on line 61 in `components/Hero.tsx`, then that should fix this issue. Then we should be good to go!